src/applications/mhv-medications/components/PrescriptionDetails/GroupedMedications.jsx

```diff
@@ -0,0 +1,105 @@
+import { VaPagination } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { waitForRenderThenFocus } from '@department-of-veterans-affairs/platform-utilities/ui';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { EMPTY_FIELD } from '../../util/constants';
+import { dateFormat, fromToNumbs } from '../../util/helpers';
+import LastFilledInfo from '../shared/LastFilledInfo';
+
+const MAX_PAGE_LIST_LENGTH = 10;
+const MAX_GROUPED_LIST_LENGTH = 26;
+const displayShowingInfoSelector = '#list-showing-info';
+
+const GroupedMedications = props => {
+  const { groupedMedicationsList } = props;
+
+  const truncatedGroupedMedicationsList = groupedMedicationsList?.slice(
+    0,
+    MAX_GROUPED_LIST_LENGTH,
+  );
+  const totalListCount = truncatedGroupedMedicationsList.length;
```

Comment:
Missing the `?.length`

Reply:
idk if thats needed since its already null checked in line 16.

Reply:
If it's needed at all, it would still be needed. That check above is just for that instance and what cause any issues but if its null or undefined here, it will throw an error